plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ansible_galaxy_install
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Install Ansible roles or collections using ansible-galaxy
+version_added: 3.4.0
+description:
+  - This module allows the installation of Ansible collections or roles using C(ansible-galaxy).
+options:
+  type:
+    description:
+    - The type of installation performed by C(ansible-galaxy).
+    - If I(type) is C(both), then I(requirements_file) must be passed and it may contain both roles and collections.
+    - "Note however that the opposite is not true: if using a I(requirements_file), then I(type) can be any of the three choices."
+    type: str
+    choices: [collection, role, both]
+    required: true
+  name:
+    description:
+    - Name of the collection or role being installed.
+    - Versions can be specified with C(ansible-galaxy) usual formats. For example, C(community.docker:1.6.1) or C(ansistrano.deploy,3.8.0).
+    - I(name) and I(requirements_file) are mutually exclusive.
+    type: str
+  requirements_file:
+    description:
+    - Path to a file containing a list of requirements to be installed.
+    - It works for I(type) equals to C(collection) and C(role).
+    - I(name) and I(requirements_file) are mutually exclusive.
+    type: path
+  dest:
+    description:
+    - The path to the directory containing your collections or roles, according to the value of I(type).
+    - >
+      Please notice that C(ansible-galaxy) will not install collections with I(type) C(both), when I(requirements_file)
+      contains both roles and collections and I(dest) is specified.
+    type: path
+  force:
+    description:
+    - Force overwriting an existing role or collection.
+    - Using I(force) as C(true) is mandatory when downgrading.
```

Comment:
Isn't it also required for upgrading, at least for older Ansible versions?

Reply:
Hmmm... Any chance you have some version number in mind? A.k.a. how far back should I look for this?

Reply:
I think only Ansible 2.9 and ansible-base 2.10 are relevant here - the UX for `ansible-galaxy collection` has been improved for ansible-core 2.11 I think.

For installing collections, you basically had to use `--force` all the time if it was already installed IIRC.

Reply:
Okie, will add that to the integration tests and see how it behaves on the different versions

Reply:
I fixed the locale issue, so now back to this one :-) From what I saw that problem only happens in 2.9, but will make sure.

Reply:
@felixfontein This behaviour you described happens, indeed, in 2.9 and 2.10. As of now, I explained that in the documentation - I didn't want to preemptively add `--force` to all the calls (for those versions), but I got mixed feelings about this. What do you think?